README.adoc

```diff
@@ -118,6 +121,20 @@ see the step doc for more documentation about he allowed versions
 infra.stashJenkinsWar("2.110")
 ----
 
+=== buildPluginWithGradle()
+
+Builds a Jenkins plugin using Gradle.
+The implementation follows the standard build/test/archive pattern. 
+The method targets compatibility with [Gradle JPI Plugin](https://github.com/jenkinsci/gradle-jpi-plugin),
+and it may not work for other use-cases.
+
+The interface is similar to `buildPlugin()`, but some features are not supported.
+Examples of not supported features:
+
+* [JEP-305](https://github.com/jenkinsci/jep/tree/master/jep/305)
+* Publushing of static analysis and coverage reports (Checkstyle, SpotBugs, JaCoCo)
```

Comment:
`Publushing` contains a typo.

Reply:
Spelling is disabled in GitHub editor :(

```suggestion
* Publishing of static analysis and coverage reports (Checkstyle, SpotBugs, JaCoCo)
```